locales/en-US/common.ftl

```diff
@@ -42,6 +42,7 @@ nav-logo-alt = Rust Logo
 footer-doc = Documentation
 footer-ask = Ask a Question on the Users Forum
 footer-status = Check Website Status
+footer-sup-doc = Supplementary Documentation
```

Comment:
I feel like this is a bit more descriptive, as a lot of people know it as forge.
```suggestion
footer-sup-doc = Rust Forge (Contributor Documentation)
```